### PR TITLE
Support fsLR-space shape files generated by preprocessing pipelines

### LINKED ISSUE #764: Add surface shape files to `init_warp_surfaces_to_template_wf`
## Summary
Nibabies (and possibly fMRIPrep) will output fsnative-space shape files (sulcal depth, sulcal curvature, and cortical thickness) that xcp-d will need to warp to fsLR-32k.

### PR DESCRIPTION
**Pull request by tsalo**
Closes #764.

## Changes proposed in this pull request

- Look for nibabies-generated surface files (`sulc.shape.gii`, `curv.shape.gii`, `thickness.shape.gii`), in fsLR space, to copy to the postprocessing derivatives.
- Create a new workflow that calls all of the anatomical workflows.

To do:

- [x] Test the new behavior.

### CODE REVIEW

File: xcp_d/tests/test_workflow_anatomical.py [hunk 1]
```diff
@@ -0,0 +1,272 @@
...
+    standard_spaces_available = {
+        "mesh": True,
+        "morphometry": True,
+        "shape": True,
+    }
+    surfaces_found = {
+        "mesh": True,
+        "morphometry": True,
+        "shape": True,
+    }
```
Comment (tsalo) [inline xcp_d/tests/test_workflow_anatomical.py:83]: @madisoth can you look through the combinations of `standard_spaces_available` and `surfaces_found` in these tests to see if they (1) are all possible and (2) cover all realistic combinations we'll get from either nibabies or fmriprep?

**Comment by tsalo** [comment]
Given that https://github.com/nipreps/fmriprep/pull/2959 has been merged, I may not need to warp the morphometric files to fsLR. I just need to pass them along to the xcp-d derivatives.

File: xcp_d/interfaces/prepostcleaning.py
Comment (mattcieslak) [inline xcp_d/interfaces/prepostcleaning.py:1]: I've been thinking about the name of this file for awhile. Can you think of a clearer name for it?

File: xcp_d/tests/test_workflows_anatomical.py [hunk 2]
```diff
@@ -1,18 +1,135 @@
-"""Tests for the xcp_d.workflows.anatomical module."""
+"""Tests for the xcp_d.workflow.anatomical module."""
```
Comment (mattcieslak) [inline xcp_d/tests/test_workflows_anatomical.py:1]: should we change this to `workflows`?

File: xcp_d/workflows/outputs.py [hunk 3]
```diff
@@ -6,10 +6,110 @@
 from niworkflows.engine.workflows import LiterateWorkflow as Workflow
 
 from xcp_d.interfaces.bids import DerivativesDataSink
+from xcp_d.interfaces.utils import FilterUndefined
 from xcp_d.utils.bids import get_entity
 from xcp_d.utils.doc import fill_doc
 
 
+@fill_doc
+def init_copy_inputs_to_outputs_wf(output_dir, name="copy_inputs_to_outputs_wf"):
```
Comment (mattcieslak) [inline xcp_d/workflows/outputs.py:15]: beautiful

**Review by mattcieslak — APPROVED** [review]

File: xcp_d/tests/test_workflows_anatomical.py (hunk 2 above)
Comment (tsalo) [reply xcp_d/tests/test_workflows_anatomical.py:1]: Good catch. Not sure how I managed to revert that.

File: xcp_d/interfaces/prepostcleaning.py
Comment (mattcieslak) [reply xcp_d/interfaces/prepostcleaning.py:1]: That is way easier to interpret. Or even `noise_handling` or something

Comment (tsalo) [reply xcp_d/interfaces/prepostcleaning.py:1]: Thanks!

**Pull request merged**